.travis.yml

```diff
@@ -37,6 +37,8 @@ matrix:
     env: CAPTURE_STDERR=true
   - rvm: jruby-19mode
     env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false -Xcli.debug=true --debug'
+  - rvm: 2.2.3
+    env: RAILS_VERSION=master
```

Comment:
(That's why there were two builds on master https://github.com/rails-api/active_model_serializers/pull/1360#commitcomment-14738951)